Examples/Swift/OHHTTPStubsDemo.xcodeproj/project.pbxproj

```diff
@@ -328,6 +333,7 @@
 				IPHONEOS_DEPLOYMENT_TARGET = 8.3;
 				MTL_ENABLE_DEBUG_INFO = NO;
 				SDKROOT = iphoneos;
+				SWIFT_OPTIMIZATION_LEVEL = "-Owholemodule";
 				SWIFT_VERSION = 2.3;
```

Comment:
Why is this line still 2.3? (Idea reading that from the GitHub web page on mobile, so hard to easily get the context of that line in the whole xcodeproj to check if it's normal, but seems strange to me)